hc/api/tests/test_admin.py

```diff
@@ -1,19 +1,24 @@
 from hc.api.models import Channel, Check
 from hc.test import BaseTestCase
+from django.contrib.auth.models import User
```

Comment:
Import order should adhere to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) standard